src/GeneralStateTestsFiller/stExample/02_tloadAfterStoreFiller.yml

```diff
@@ -0,0 +1,61 @@
+# 02
+# Loading after storing returns the stored value: TSTORE(x, y), TLOAD(x) returns y
+#
+# Expect storage slot 1 to have value 88
+
+02_tloadAfterStore:
```

Comment:
tload after tstore 

tload after store would be a case to check that store does not affect tload on the same address

Reply:
right, updated this test and added a new tests [#18 ](https://github.com/snreynolds/tests/pull/1/commits/484f3d410b34c884392a2c1a4a5f7c78f3eea095#diff-423a30eb498a4ce2be8ca8929da3d6efdb52425b53f05d3f49d467fc5402a409R6)